bccsp/factory/opts.go

```diff
@@ -25,6 +24,10 @@ func GetDefaultOpts() *FactoryOpts {
 			Hash:     "SHA2",
 			Security: 256,
 		},
+		PKCS11: &pkcs11.PKCS11Opts{
```

Comment:
I guess it's fine to consolidate these into a single struct regardless of build tag because we never try to interact with the pkcs11 lib unless we follow the path set out in the pkcs11.go file?

Reply:
Yes, that was the thinking here.